functions/src/models/account.model.ts

```diff
@@ -0,0 +1,22 @@
+export interface Account {
+  displayName: string;
+  photoUrl: string;
+  creationAt: string;
+  lastSignInAt: string;
+  credentials: {
+    accessToken: string;
+    provider: string;
+    refreshToken?: string;
+    secret?: string;  // Optional field
```

Comment:
I don't think the comment `optional field` is needed because the other optional fields don't have any mention and it is clear with the `?`